gatsby/src/pages/topics.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { graphql, PageRendererProps, Link } from "gatsby";
+import { Topic } from "../types";
+import { Layout } from "../components/Layout/Layout";
+import { SEO } from "../components/SEO/SEO";
+
+type TopicPageProps = {
+	data: {
+		allTopics: {
+			nodes: Topic[];
```

Comment:
Shouldn't this be a PartialTopic?
```suggestion
			nodes: PartialTopic[];
```

Reply:
I was a little unsure since I assumed we would be using the topic summary property on this page so went with the entire topic. Should I change it?

Reply:
I see, we don't on live currently. The problem with it as is as it has all the other properties on the typescript type, but the data isn't there as those other fields aren't in the query, so we should change it